### PR TITLE
Improve occupancy testing cluster tests

### PR DESCRIPTION
**Pull request by fessehaeve**
#### Problem
Some OCC tests are dummy tests

#### Change overview
Improve occupancy testing cluster tests

#### Testing
chip-tool tests

### CODE REVIEW

**Review by woody-apple — APPROVED** [review]
@raju-apple can you please review?

**Review by kvikrambhat — APPROVED** [review]

**Review by raju-apple — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]
Approved, but these are marked as "manual tests" right now.  Should they be moved into the automated tests set so they actually run in CI?

**Comment by fessehaeve** [comment]
> Approved, but these are marked as "manual tests" right now. Should they be moved into the automated tests set so they actually run in CI?

I honestly don't know the procedure. The people responsible for this should pitch in here? @woody-apple

**Comment by bzbarsky-apple** [comment]
Let me rephrase: if the test passes in CI, and is not extremely time-consuming, it should run in CI.

**Review by andy31415 — APPROVED** [review]
